Description: Revert the revert of async rename
 This patch reverts https://review.openstack.org/#/c/597421/ which itself
 reverts the rename of the async keyword. Upstream did that because they
 had to maintain compat with older oslo.db.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2018-09-25

diff --git a/nova/db/sqlalchemy/api.py b/nova/db/sqlalchemy/api.py
index a67747e408..59f0a8c292 100644
--- a/nova/db/sqlalchemy/api.py
+++ b/nova/db/sqlalchemy/api.py
@@ -202,7 +202,7 @@ def select_db_reader_mode(f):
         use_slave = keyed_args.get('use_slave', False)
 
         if use_slave:
-            reader_mode = get_context_manager(context).async
+            reader_mode = get_context_manager(context).async_
         else:
             reader_mode = get_context_manager(context).reader
 
